docs/node/extensions/oracles/Deposits.mdx

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 3
+sidebar_label: "Eth Deposits"
+id: eth-deposits-oracle
+title: Eth Deposits Oracle
+description: Using the Eth Deposits Oracle
+slug: /extensions/oracles/eth-deposits
+---
+
+In this tutorial, we will learn how to use the Eth Deposits Oracle to acquire Kwil gas.  `eth-deposits` is a built-in oracle extension that can be used to credit accounts on a Kwil network in response to events emitted on the Ethereum network. This oracle actively listens to the events of signature `Credit(address,uint256)` emitted by a specific contract address and  credits the accounts on the Kwil network upon attestation by a super-majority (two-thirds) of Validators confirming the occurrence of this event.
+
+Therefore these smart contracts can be used in a variety of use cases, such as:
+- Purchasing Kwil gas by depositing ERC20 tokens into a smart contract on Ethereum, with the tokens being securely held in the smart contract.
+- Minters of NFTs can be rewarded with Kwil gas as a part of the minting process.
+- Users can acquire Kwil gas by burning ERC20 tokens on Ethereum.
+
+Follow the steps below to use the Eth Deposits Oracle:
+1. [Implement the Smart contract](#implement-the-smart-contract) that emits the `Credit(address,uint256)` event.
+2. [Deploy the smart contract](#deploy-the-smart-contract) on an EVM chain.
+3. [Configure Kwil node](#configure-kwil-node) to start the `eth-deposits` oracle.
+4. [Credit accounts on the Kwil network](#credit-accounts-on-the-kwil-network) through `eth-deposits` oracle.
+
+## Implement The Smart Contract
+
+The first step on using the `eth-deposits` oracle is to implement a smart contract with methods that emits the `Credit(address,uint256)` event. The `Credit` event should contain the address of the user and the amount of tokens deposited. 
+
+Below is an example of a smart contract that emits the `Credit(address,uint256)` event whenever ERC20 tokens are deposited into the smart contract using the `deposit` method. This is just a sample example showing the usage of the `Credit` event and by no means a complete implementation of the smart contract. 
+
+```solidity
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract EthDeposits {
+    IERC20 public token;
+
+    // Credit event containing the address of the user and the amount of tokens deposited
+    // This event will be listened by the eth-deposits oracle
+    event Credit(address indexed user, uint256 amount);
+
+    constructor(address _token) {
+        token = IERC20(_token);
```

Comment:
can you remove the IERC20s and such? Literally just make it as simple as possible